pytx3/pytx3/cli/fetch2.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import requests
+import typing as t
+from .. import TE
+from ..dataset import Dataset
+from . import command_base
+
+class Fetch2Command(command_base.Command):
+    """
+    Download content from ThreatExchange to disk.
+
+    Using the CollaborationConfig, identify ThreatPrivacyGroup that
+    corresponds to a single collaboration and fetch related threat updates.
+    """
+
+    @classmethod
+    def init_argparse(cls, ap) -> None:
+        ap.add_argument(
+            "--start-time",
+            type=int,
+            help="Fetch updates that occured on or after this timestamp",
+        )
+        ap.add_argument(
+            "--stop-time",
+            type=int,
+            help="Fetch updates that occured before this timestamp",
+        )
+        ap.add_argument(
+            "--owner",
+            type=int,
+            help="Only fetch updates for indicators that the given app has a descriptor for",
```

Comment:
Can we change the comments here and across pytx to use less specific TE terminology and more general terminology for how we think about the data model. Some examples:
"Indicators" -> "Threats"
"descriptors" -> "opinions"
"privacy group" -> "collaboration"

Here i'd like to see something more like: "Only fetch updates for threats which the given app has added opinions about.

Reply:
I like the direction of changes to terminology, but I think these should happen first on the ThreatExchange api side. To prevent cognitive mismatch when comparing the pytx3 documentation to the ThreatExchange documentation, I think it may be good to keep the existing terminology and then overall them together (or around the same time).

Just my thought.

Reply:
@jeberl are you ok with punting on the terminology update as @bodnarbm mentioned?

Reply:
I should note that my comment was also specific to across pytx3. For this command, given it's intended use, I'm less concerned about the changes. And I'm fine either way, but would prefer to defer the terminology change across pytx3 until we are ready to make that change in the API layer (to prevent potential churn in exact wording).

Reply:
Ok. I'm taking a documentation refresh on as a project this half so can come back and make these edits